pkg/cmd/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"visualization-api/pkg/logging"
+)
+
+// var log_file *os.File
+
+func main() {
+
+	/*
+		APP INITIALIZATION STEPS
+		1 - read config file
+		2 - get log_file descriptor using path in config file, defer file close
+			to the end of main function, file descriptor should also be closed
+			on signals
+		3 - initialize logging module
+		4 - run application logic
+	*/
+	log.InitializeLogger()
```

Comment:
Not very obvious.
Looks like logging initialization, but actually config parsing, logging initialization and even watching for process signals.